Ronin.rte/Devices/Weapons/Model590/Model590.lua

```diff
@@ -1,8 +1,9 @@
 function Create(self)
 	self.pullTimer = Timer();
-
 	self.loaded = false;
 	self.rotFactor = math.pi;
+	
+	self.cockSound = CreateSoundContainer("Ronin Model 590 Cock Sound", "Ronin.rte");
```

Comment:
This is a bit of an arbitrary spot to mention this (and it's a pretty arbitrary thing to mention), but I wonder if you there's any reason to specify Ronin in the presetname, given it's in the datamodule. Maybe it's helpful for ini to be specific though, I dunno.

Reply:
Blame Max for adding faction names to each gotdam object :p

Reply:
Haha fair point. Do what works for you guys :)